### PR TITLE
 added Submit page

### PR DESCRIPTION
**Pull request by ha77an**
#27

### CODE REVIEW

File: client/src/components/App/index.js [hunk 1]
```diff
@@ -85,27 +108,20 @@ export default class App extends React.Component {
       '/submit',
       '/myprofile',
     ];
+    const staticRoutes = ['/', '/login', '/signup'];
     return (
       <div>
-        <AppBar logoutHandler={this.logoutHandler} auth={user || admin} />
+        <AppBar
+          logoutHandler={this.logoutHandler}
+          auth={user || admin}
```
Comment (Asmaathabet) [inline client/src/components/App/index.js:116]: applicant part only for users not admins. admin won't enter any application

**Comment by Asmaathabet** [comment]
@ha77an  solve conflict, please

File: client/src/components/App/index.js (hunk 1 above)
Comment (ha77an) [reply client/src/components/App/index.js]: exactly, just I thought we used the app bar in admin routes

Comment (Asmaathabet) [reply client/src/components/App/index.js]: no problem, its just a quick note

**Pull request merged**